Temporary patch to disable mono until new gtk-sharp is in tree wrt bug #382491

--- bindings/Makefile.am
+++ bindings/Makefile.am
@@ -1,9 +1,5 @@
 SUBDIRS = 
 
-if HAS_MONO
-SUBDIRS += mono
-endif
-
 if HAS_PYTHON
 SUBDIRS += python
 endif
--- configure.ac
+++ configure.ac
@@ -124,61 +124,6 @@
 AC_PATH_PROG([VALA_API_GEN], [vapigen])
 
 ###########################
-# Check for Mono support
-###########################
-
-MONO_REQUIRED_VERSION=1.0
-PKG_CHECK_MODULES(MONO_DEPENDENCY, mono >= $MONO_REQUIRED_VERSION, has_mono=true, has_mono=false)
-
-AC_PATH_PROG(AL, al, no)
-if test "x$AL" = "xno" ; then
-   AC_MSG_NOTICE([No al tool found.  You need to install Mono.])
-fi
-
-if test "x$has_mono" = "xtrue" ; then
-   GENERATED_SOURCES=generated/*.cs
-   AC_PATH_PROG(RUNTIME, mono, no)
-
-   if test "x$RUNTIME" != "no" ; then
-      RUNTIME=mono
-   fi
-
-   AC_PATH_PROG(CSC, gmcs, no)
-   LIB_PREFIX=.so
-   LIB_SUFFIX=
-fi
-
-if test "x$CSC" = "xno" ; then
-   AC_MSG_NOTICE([No Mono compiler found.])
-fi
-
-AC_PATH_PROG(GACUTIL, gacutil, no)
-if test "x$GACUTIL" = "xno" ; then
-   AC_MSG_NOTICE([No gacutil tool found])
-fi
-
-AC_SUBST(RUNTIME)
-AC_SUBST(CSC)
-AC_SUBST(GACUTIL)
-AC_SUBST(LIB_PREFIX)
-AC_SUBST(LIB_SUFFIX)
-AC_SUBST(GENERATED_SOURCES)
-
-GLIB_SHARP_REQ_VERSION=2.12.1
-GTK_SHARP_REQ_VERSION=2.12
-PKG_CHECK_MODULES(GTK_SHARP, gtk-sharp-2.0 >= $GTK_SHARP_REQ_VERSION, has_gtksharp=true, has_gtksharp=false)
-AC_SUBST(GTK_SHARP_LIBS)
-AC_SUBST(GTK_SHARP_CFLAGS)
-
-PKG_CHECK_MODULES(GAPI, gapi-2.0 >= $GLIB_SHARP_REQ_VERSION, has_gapi=true, has_gapi=false)
-AC_PATH_PROG(GAPI_PARSER, gapi2-parser, no)
-AC_SUBST(GAPI_PARSER)
-AC_PATH_PROG(GAPI_CODEGEN, gapi2-codegen, no)
-AC_SUBST(GAPI_CODEGEN)
-AC_PATH_PROG(GAPI_FIXUP, gapi2-fixup, no)
-AC_SUBST(GAPI_FIXUP)
-
-###########################
 # Check for Python support
 ###########################
 
